src/gui/attributetable/qgsdualview.cpp

```diff
@@ -116,6 +116,9 @@ QgsDualView::~QgsDualView()
 {
   QgsSettings settings;
   settings.setValue( QStringLiteral( "/qgis/attributeTable/splitterState" ), mConditionalSplitter->saveState() );
+
+  if ( mFeatureSelectionManager && mFeatureSelectionManager->parent() == this )
+    delete mFeatureSelectionManager;
```

Comment:
Is this not already done by Qt (QObject parent/child ownership)?

Reply:
Ehm yes. So I'll close this :)